core/domain/email_manager.py

```diff
@@ -263,7 +308,20 @@ def _send_bulk_mail_in_transaction(instance_id=None):
 
 
 def send_mail_to_admin(email_subject, email_body):
-    """Sends email to admin."""
+    """Send email to admin.
+
+    The email is sent to the ADMIN_EMAIL_ADDRESS set in config.
+
+    Args:
+        email_subject: str. Subject of the email.
+        email_body: str. Body (message) of the email.
+
+
+    Raises:
+        Exception: Exceptions from _send_email, send_mail such as
```

Comment:
Hi @Oishikatta -- just to follow up on this, I chatted with the other tech leads, and we all agree that the docstring for exceptions should only cover exceptions that are raised by the function itself. Sorry again for taking you in the wrong direction at the outset, and thank you very much for diving deeper on this!